### PR TITLE
Supriyo

### PR DESCRIPTION
**Pull request by supriyo-bhakat-comprinno**
I have created the following checks as part of the project. This pull request is for your review and merge. Below is the list of the checks that I have implemented:

NetworkFirewall::MultiAZEnabled
RDS::InstanceStorageEncrypted
RDS::InstanceIntegrationCloudWatchLoggingEnabled
RDS::InstanceMinorVersionUpgradeEnabled
Route53::DomainsPrivacyProtectionEnabled
S3::BucketDefaultEncryption
S3::ObjectVersioning
S3::BucketSecureTransportPolicy
S3::BucketACLProhibited
S3::BucketNoMFADelete
SecretsManager::AutomaticRotationEnabled
SecurityHub::Enabled
SNS::TopicKMSEncryptionAtRestEnabled
SSM::DocumentSecretsPresent
SSM::ManagedCompliantPatching
SSM::PatchManagerEnabled
SSM::EC2InstanceManagedBySSM
SSM::EC2InstanceManagedBySSM
VPC::FlowLogsEnabled
VPC::FlowLogsEnabled
VPC::DefaultSecurityGroupClosed
VPC::EndpointServicesAllowedPrincipalsTrustBoundaries
VPC::SecurityGroupPortRestrictionCheck
VPC::ServiceVPCEndpointEnabled
VPC::FlowLogsEnabled
WellArchitectedTool::WorkloadNoHighOrMediumRisks
CloudFront::WAFProtectionEnabled
CloudTrail::Enabled
CloudTrail::S3BucketAccessLoggingEnabled
CloudTrail::CloudWatchLoggingEnabled
CloudTrail::MultiRegionEnabled
CloudWatch::LogMetricFilterRootUsage
CloudWatch::LogMetricFilterSecurityGroupChanges
CloudWatch::LogMetricFilterUnauthorizedAPICalls
CloudWatch::LogMetricFilterPolicyChanges
CloudWatch::LogMetricFilterVPCAlarmConfigured
CloudWatch::LogMetricFilterAuthenticationFailures
-------------------------------------------------------------------------------
I have added Regular Expression for some cloudwatch checks for proper validation. will remove it if not needed

### CODE REVIEW

**Comment by abhay-pai-comprinno** [comment]
@supriyo-bhakat-comprinno - Kindly update your branch i.e. `supriyo` with the `main` branch. Ping me if you need help.

**Comment by abhay-pai-comprinno** [comment]
@supriyo-bhakat-comprinno - Kindly remove/comment all `print` statements from this PR.

**Comment by supriyo-bhakat-comprinno** [comment]
> @supriyo-bhakat-comprinno - Kindly remove/comment all `print` statements from this PR.

Sure .Noted

**Comment by supriyo-bhakat-comprinno** [comment]
> @supriyo-bhakat-comprinno - Kindly update your branch i.e. `supriyo` with the `main` branch. Ping me if you need help.

Sure. will do that

**Review by abhay-pai-comprinno — APPROVED** [review]

**Review by abhay-pai-comprinno — APPROVED** [review]

**Review by abhay-pai-comprinno — APPROVED** [review]

File: library/aws/checks/iam/root_mfa_enabled.py [hunk 1]
```diff
@@ -9,27 +8,33 @@
...
             account_summary = client.get_account_summary()
             
-            # Extract the value indicating if root MFA is enabled
+           
             root_mfa_enabled = account_summary['SummaryMap']['AccountMFAEnabled']
 
-            # Check if MFA is enabled for the root user
+            
             if root_mfa_enabled == 1:
+                print("pass")
```
[the remark this answers is not in the document]
Comment (supriyo-bhakat-comprinno) [reply library/aws/checks/iam/root_mfa_enabled.py:28]: removed

**Pull request merged**